src/rabbitstressor.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
More specifically, if the stressor is moved to `bin/`, it will no longer get executed by the unit tests. Would it make sense to compile the code under `bin/` over to `lib/` as well? I am trying to figure out the best place for the stressor because it is just a convenience tool to add custom stress to queues. Eventually, the plan is to have it go under its own repo from my understanding. But for the moment, I think it needs a temporary home with tests :)